test/unit/vdom/skip.js

```diff
@@ -0,0 +1,60 @@
+import { define, prop, props, symbols, vdom } from '../../../src/index';
+import afterMutations from '../../lib/after-mutations';
+import fixture from '../../lib/fixture';
+
+describe('vdom/skip', () => {
+  it('should skip the element children', done => {
+    const Elem = define('x-test', {
+      props: {
+        num: prop.number()
+      },
+      render() {
+        vdom.element('div', { skip: true }, () => {
+          vdom.element('span', () => {
```

Comment:
Yeah, this gets kinda complicated.

- Incremental DOM allows you to put content in it, you just have to either call skip(), or generate content.
- Once you call `skip()`, it only throws if you're using the untranspiled source.
- If you're using the `dist/` (which most people are), it doesn't throw. It just does nothing.

This is really just syntactic sugar for doing what you can already do with Incremental DOM, it just removes the need to know about implementation details where you must either call `skip()` or generate content. In our case, you can switch skipping on and off and it will manage either calling `skip()` or rendering your descendant tree.